### PR TITLE
[Update] How to Initiate a Cross Data Center Migration for your Linode

### PR DESCRIPTION
**Pull request by leslitagordita**
* Updated guide gif to display latest Cloud Manager UI.
* Added new section showing how to initiate a migration from the Linodes landing page.
* Update guide URL to use our retitling rubric and other front matter SEO updates.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 4f0698d1a95c5ba1eaa1257a8d86e2e5300f84f8

https://deploy-preview-3370--nostalgic-ptolemy-b01ab8.netlify.app

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 93ce4f8f2d8a9a83d6b4cbc66507ae096f831994

https://deploy-preview-3370--nostalgic-ptolemy-b01ab8.netlify.app

File: docs/platform/disk-images/how-to-initiate-a-cross-data-center-migration-for-your-linode/index.md [hunk 1]
```diff
@@ -18,7 +21,7 @@ You can migrate your Linode across data centers using the Linode Cloud Manager.
 We recommend using our [Choosing a Data Center](/docs/platform/how-to-choose-a-data-center/) guide to learn how to choose and speed test a data center.
 {{< /note >}}
 
-This guide will cover:
+## In this Guide:
 
 - [Important details to know before initiating a cross data center migration](/docs/platform/migrating-to-a-different-data-center/#things-to-know-before-migrating).
```
Comment (andystevensname) [inline docs/platform/disk-images/how-to-initiate-a-cross-data-center-migration-for-your-linode/index.md:26]: Can these just be shortened to anchors? Or, update the links to point to the new guide URL slug?

Comment (leslitagordita) [reply docs/platform/disk-images/how-to-initiate-a-cross-data-center-migration-for-your-linode/index.md]: Updated

**Pull request merged**